docs/rest-api/getting-started.md

```diff
@@ -0,0 +1,92 @@
+# Getting started with the REST API
+
+The REST API is a powerful part of WooCommerce which lets you read and write various parts of WooCommerce data such as orders, products, coupons, customers, and shipping zones.
+
+## Requirements
+
+In order to access the REST API using the standard endpoint URI structure (e.g. `wc/v3/products`), you must have your WordPress permalinks configured to something other than "Plain". Go to **Settings > Permalinks** and choose an option.
+
+![Permalinks options](_media/permalinks.webp)
+
+## API reference
+
+[WooCommerce REST API Docs](https://woocommerce.github.io/woocommerce-rest-api-docs/) provides technical details and code samples for each API endpoint.
+
+## Authentication
+
+Authentication is usually the part most developers get stuck on, so this guide will cover a quick way to test that your API is working on your server and you can authenticate. **If this works, but your code to use the API does not, it means there's a problem with your code. Please do not open issues asking for support about this on Github - use the support forums.**
+
+We'll use both [Postman](https://www.getpostman.com/) and [Insomnia](https://insomnia.rest/) clients in these examples. Both are free and will help you visualise what the API offers.
+
+Before proceeding, please read the [REST API docs on authentication which covers the important parts concerning API Keys and Auth](https://woocommerce.github.io/woocommerce-rest-api-docs/#authentication). We're only covering connecting over HTTPS here since it's the simplest and most secure method. You should avoid HTTP if possible.
+
+## Generate Keys
+
+To start using REST API, you first need to generate API keys.
+
+1. Go to *WooCommerce > Settings > Advanced*
+2. Go to the *REST API* tab and click *Add key*.
+3. Give the key a description for your own reference, choose a user with access to orders etc, and give the key *read/write* permissions.
+4. Click *Generate api key*.
+5. Your keys will be shown - do not close this tab yet, the secret will be hidden if you try to view the key again.
+
+![Generated API Keys](_media/keys.png)
+
+## Make a basic request
+
+The request URL we'll test is `wp-json/wc/v3/orders`. On localhost the full URL may look something like this: `https://localhost:8888/wp-json/wc/v3/orders`. Modify this to use your own site URL.
+
+In Postman, you need to set the fields for request type, request URL, and the settings on the authorization tab. For Authorization, choose *basic auth* and enter your *consumer key* and *consumer secret* keys from WooCommerce into the username and password fields
+
+Once done, hit send and you'll see the JSON response from the API if all worked well. You should see something like this:
+
+![Generated API Keys](_media/postman.png)
```

Comment:
I guess it would be good to update the screenshots to show the proper endpoint to avoid confusion (as they are currently showing the v1 version of the endpoint), but definitely not critical.

Reply:
I'm not super concerned about showing an old endpoint URI, since the point of the screenshot is more about how to use Postman and Insomnia. As far as I know the UI for those two apps is still basically the same (although I don't really use Postman), so I'm inclined to leave it for now.